R/scalar_in_linter.R

```diff
@@ -28,14 +30,16 @@
 #' @evalRd rd_tags("scalar_in_linter")
 #' @seealso [linters] for a complete list of linters available in lintr.
 #' @export
-scalar_in_linter <- function() {
+scalar_in_linter <- function(add_in_operators = "%chin%") {
   # TODO(#2085): Extend to include other cases where the RHS is clearly a scalar
   # NB: all of logical, integer, double, hex, complex are parsed as NUM_CONST
-  xpath <- "
-  //SPECIAL[text() = '%in%' or text() = '%chin%']
+  special <- paste(paste0("text() = '", c("%in%", add_in_operators), "'"), collapse = " or ")
+
+  xpath <- paste0("
```

Comment:
we are usually using `glue()` once an XPath is parameterized, e.g.

```r
XPath <- glue::glue("
  //SPECIAL[{ special }]
    ...
")
```

Reply:
And now I use `glue` as well